bundles/org.openhab.automation.pythonscripting/README.md

```diff
@@ -0,0 +1,574 @@
+| postUpdate               | postUpdate(state)                                                                     |                                                                                                     |
+| postUpdateIfDifferent    | postUpdateIfDifferent(state)                                                          |                                                                                                     |
+| sendCommand              | sendCommand(command)                                                                  |                                                                                                     |
+| sendCommandIfDifferent   | sendCommandIfDifferent(command)                                                       |                                                                                                     |
+| getPersistence           | getPersistence(service_id = None)                                                     | [ItemPersistence](#class-itempersistence)                                         |
+| getSemantic              | getSemantic()                                                                         | [ItemSemantic](#class-itemsemantic)                                               |
+| <...>                    | see [openHAB Item api](https://www.openhab.org/javadoc/latest/org/openhab/core/items/item) |                                                                                                |
+
+### class GroupItem 
+
+GroupItem is an extended [Item](#class-item) which wraps results from getAllMembers & getMembers into [Items](#class-item)
+
+### class ItemPersistence 
+
+ItemPersistence is a wrapper around [openHAB PersistenceExtensions](https://www.openhab.org/javadoc/latest/org/openhab/core/persistence/extensions/persistenceextensions). The parameters 'item' and 'serviceId', as part of the Wrapped Java API, are not needed, because they are inserted automatically.
+
+| Function                 | Usage                                                                                 | Description                                                                                         |
+| ------------------------ | ------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------- |
+| getStableMinMaxState     | getStableMinMaxState(time_slot, end_time = None)                                      | Average calculation which takes into account the values ​​depending on their duration                 |
+| getStableState           | getStableState(time_slot, end_time = None)                                            | Average calculation which takes into account the values ​​depending on their duration                 |
```

Comment:
Why the zero width spaces?

```suggestion
| getStableMinMaxState     | getStableMinMaxState(time_slot, end_time = None)                                      | Average calculation which takes into account the values depending on their duration                 |
| getStableState           | getStableState(time_slot, end_time = None)                                            | Average calculation which takes into account the values depending on their duration                 |
```

Reply:
![grafik](https://github.com/user-attachments/assets/f09bb80b-55bf-47d2-924b-afb959f7f796)

it is just to align table column separator which makes it easier to read during edit.